packages/react/src/components/Dropdown/Dropdown.js

```diff
@@ -24,304 +16,263 @@ const defaultItemToString = item => {
+    helperText,
+    translateWithId,
+    light,
+    invalid,
+    invalidText,
+    // mapDownshiftProps
+  } = props;
+
+  const {
+    isOpen,
+    selectedItem,
+    getToggleButtonProps,
+    getLabelProps,
+    getMenuProps,
+    getItemProps,
+    highlightedIndex,
+  } = useSelect({ items });
+
+  const inline = type === 'inline';
+
+  const className = ({ isOpen }) =>
+    cx(`${prefix}--dropdown`, containerClassName, {
+      [`${prefix}--dropdown--invalid`]: invalid,
+      [`${prefix}--dropdown--open`]: isOpen,
+      [`${prefix}--dropdown--inline`]: inline,
+      [`${prefix}--dropdown--disabled`]: disabled,
+      [`${prefix}--dropdown--light`]: light,
+      [`${prefix}--dropdown--${size}`]: size,
     });
-    const helper = helperText ? (
-      <div id={helperId} className={helperClasses}>
-        {helperText}
-      </div>
-    ) : null;
-    const wrapperClasses = cx(
-      `${prefix}--dropdown__wrapper`,
-      `${prefix}--list-box__wrapper`,
-      {
-        [`${prefix}--dropdown__wrapper--inline`]: inline,
-        [`${prefix}--list-box__wrapper--inline`]: inline,
-        [`${prefix}--dropdown__wrapper--inline--invalid`]: inline && invalid,
-        [`${prefix}--list-box__wrapper--inline--invalid`]: inline && invalid,
-      }
-    );
-
-    // needs to be Capitalized for react to render it correctly
-    const ItemToElement = itemToElement;
-    return (
-      <div className={wrapperClasses}>
-        {title}
-        {!inline && helper}
-        <Downshift
-          {...downshiftProps}
-          onChange={this.handleOnChange}
-          itemToString={itemToString}
-          defaultSelectedItem={initialSelectedItem}
-          selectedItem={selectedItem}>
-          {({
-            isOpen,
-            itemToString,
-            selectedItem,
-            highlightedIndex,
-            getRootProps,
-            getButtonProps,
-            getItemProps,
-            getLabelProps,
-            toggleMenu,
-          }) => {
-            const buttonProps = {
-              ...getButtonProps({
-                onKeyDown: event => {
-                  if (match(event, keys.Enter)) {
-                    toggleMenu();
+
+  const titleClasses = cx(`${prefix}--label`, {
+    [`${prefix}--label--disabled`]: disabled,
+  });
+
+  const helperClasses = cx(`${prefix}--form__helper-text`, {
+    [`${prefix}--form__helper-text--disabled`]: disabled,
+  });
+
+  const wrapperClasses = cx(
+    `${prefix}--dropdown__wrapper`,
+    `${prefix}--list-box__wrapper`,
+    {
+      [`${prefix}--dropdown__wrapper--inline`]: inline,
+      [`${prefix}--list-box__wrapper--inline`]: inline,
+      [`${prefix}--dropdown__wrapper--inline--invalid`]: inline && invalid,
+      [`${prefix}--list-box__wrapper--inline--invalid`]: inline && invalid,
+    }
+  );
+
+  // needs to be Capitalized for react to render it correctly
+  const ItemToElement = itemToElement;
+
+  const handleOnChange = ({ onChange }, selectedItem) => {
```

Comment:
Style-nit: prefer function declarations to arrow functions for handlers

```suggestion
  function handleOnChange({ onChange }, selectedItem) {
```